docs/user-guide/installation.md

```diff
@@ -14,3 +14,19 @@ conda install -c conda-forge -c scipp essreflectometry
 ```
 ````
 `````
+
+To install ESSreflectometry and all of its dependencies,
+including dependencies used in the graphical user interface for batch reduction, use
+
+`````{tab-set}
+````{tab-item} pip
+```sh
+pip install essreflectometry[all]
+```
+````
+````{tab-item} conda
+```sh
+conda install -c conda-forge -c scipp essreflectometry[all]
```

Comment:
unfortunately the optional deps `[all]` syntax doesn't work with conda, so it needs to be updated to be include the extra deps manually.

Reply:
```suggestion
conda install -c conda-forge -c scipp essreflectometry ipywidgets ipydatagrid ipytree pythreejs
```
Yes, extra installs don't exist in conda, so we either need to package up the essreflectometry conda package with everything in `all` as dependencies or manually install the extra dependencies.